runtime/expr/cutter.go

```diff
@@ -67,30 +55,42 @@ func (c *Cutter) FoundCut() bool {
 // receiver's configuration.  If the resulting record would be empty, Apply
 // returns zed.Missing.
 func (c *Cutter) Eval(ectx Context, in *zed.Value) *zed.Value {
+	rb, paths, verr := c.lookupBuilder(ectx, in)
+	if verr != nil {
+		if !verr.IsMissing() && !verr.IsQuiet() {
+			verr = ectx.CopyValue(*c.zctx.WrapError(fmt.Sprintf("cut: %s", zed.DecodeError(verr.Bytes())), in))
```

Comment:
`zed.DecodeError` is broken. See #4810.

Reply:
What do you propose to do instead here?

Reply:
Just decode it as a string?

Reply:
We're just dropping the wrap for now and will revisit this...

Reply:
OK, here's my proposal.

1. lookupBuilder should return a Go error instead of a zed.Value.
2. lookupBuilder should convert a Zed error to a Go error like this.
   ```go
   	for _, lval := range c.lvals {
   		path, errVal := lval.Eval(ectx, in)
   		if errVal != nil {
   			val := zed.NewValue(errVal.Type.(*zed.TypeError).Type, errVal.Bytes())
   			if val.IsString() {
   				return errors.New(val.AsString())
   			}
   			return errors.New(zson.FormatValue(val))
   		}
   		...
   	}
   ```
3. Cutter.Eval should handle an error from lookupBuilder like this.
   ```go
   	b, paths, err := c.lookupBuilder(ectx, in)
   	if err != nil {
   		return ectx.CopyValue(*c.zctx.WrapError(fmt.Sprintf("cut: %s", err, in))
   	}
   ```
4. We should create an issue to revisit this.

Reply:
Not great but an improvement to what I have.